### ISSUE TITLE
wrong solution for "flex/03-flex-header-2" exercise

### ISSUE THREAD

**Issue by benedictfernando** [TheOdinProject/css-exercises#121]
### issue: 

- rightmost header components _(button and notification)_ are **not aligned** with the desired outcome

### solution link: 

- https://github.com/TheOdinProject/css-exercises/blob/main/flex/03-flex-header-2/solution/solution.css, _line 50 and beyond_

**Comment by thatblindgeye** [comment]
Can you explain what you mean by those components not being "aligned with the desired outcome"?

**Comment by benedictfernando** [comment]
When [solution.css](https://github.com/TheOdinProject/css-exercises/blob/main/flex/03-flex-header-2/solution/solution.css) is linked in pair with the [solution.html](https://github.com/TheOdinProject/css-exercises/blob/main/flex/03-flex-header-2/solution/solution.html), the `profile-image` and `notifications` elements come up around the middle part of the navbar instead of it being at the leftmost corner.

**Comment by thatblindgeye** [comment]
I'm still unsure what the issue is that you're referring to. When I open the solution.html file locally, it looks and behaves as the Desired Outcome of the README.

If you're seeing something different it would help if you provided an image and/or made sure your solution.html and solution.css files weren't altered.

**Comment by thatblindgeye** [comment]
@benedictfernando Checking in to see if you've seen the above message.

**Comment by benedictfernando** [comment]
Hello @thatblindgeye~ apologies for my mistake. I have reckoned it's an oversight on my side, since I have mixed and matched the `solution.css` file from the repository to my own version of `index.html`, forgetting to use `solution.html` in pair with it. Furthermore, upon testing it in a Codepen, you're definitely right that the solution behaves accordingly with the _desired outcome_ of the README.

Much apologies for these once more; but many thanks for addressing the issue regardless~ I genuinely appreciate it @thatblindgeye, thank you so much!
